src/mgmt/srv_target.c

```diff
@@ -1123,7 +1122,12 @@ ds_mgmt_hdlr_tgt_create(crt_rpc_t *tc_req)
 
 	tgt_scm_sz  = tc_in->tc_scm_size / dss_tgt_nr;
 	tgt_meta_sz = tc_in->tc_meta_size / dss_tgt_nr;
-	vos_pool_roundup_size(&tgt_scm_sz, &tgt_meta_sz);
+	rc          = vos_pool_roundup_size(&tgt_scm_sz, &tgt_meta_sz);
```

Comment:
Is this strange formatting enforced by the clang linter?

Reply:
yes